cmd/init.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2017 The Kedge Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"html/template"
+
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	DefaultFileName = "kedge.yml"
+)
+
+var (
+	FileName, image, name string
```

Comment:
Why is `FileName` public but `image` and `name` private?
Shouldn't be everything private? Those variables are not supposed to be used outside this packed, or are they?